.github/workflows/workflow.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
           python -m poetry lock
           python -m poetry show --tree
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           directory: ./coverage/reports/
```

Comment:
```suggestion
```

Reply:
@thomasrockhu-codecov : I applied the change and re-reran the GitHub action and it still failed due to the same error.